app/assets/javascripts/static_pages.js

```diff
@@ -1 +1,219 @@
 
+// $(document).ready(function(){
```

Comment:
What is the other issue

Btw clement doesn't get notifications
On 17 Feb 2015 23:28, "Harry Chen" <notifications@github.com> wrote:

> In app/assets/javascripts/static_pages.js
> <https://github.com/cchchoi1986/hikehk/pull/23#discussion_r24822969>:
>
> > @@ -1 +1,219 @@
> >
> > +// $(document).ready(function(){
>
> I have only fixed the gitignore issues. Please fix other issues. Cassie
> Mak <notifications@github.com>于2015年2月17日星期二写道：
> … <#14b98273c3804542_>
>  In app/assets/javascripts/static_pages.js <
> https://github.com/cchchoi1986/hikehk/pull/23#discussion_r24822273>: > @@
> -1 +1,219 @@ > > +// $(document).ready(function(){ You said you fixed it?
> So merge after you fixed it. Then I will pull request On 17 Feb 2015 23:14,
> "Harry Chen" <notifications@github.com <javascript:_e(%7B%7D,'cvml','
> notifications@github.com');>> wrote: … <#14b9820745c77980_> In
> app/assets/javascripts/static_pages.js <
> https://github.com/cchchoi1986/hikehk/pull/23#discussion_r24821810>: > @@
> -1 +1,219 @@ > > +// $(document).ready(function(){ What do you mean? —
> Reply to this email directly or view it on GitHub <
> https://github.com/cchchoi1986/hikehk/pull/23/files#r24821810>. — Reply
> to this email directly or view it on GitHub <
> https://github.com/cchchoi1986/hikehk/pull/23/files#r24822273>.
>  -- *Harry Chen* Website <http://www.harrychen.net> | LinkedIn <
> http://ca.linkedin.com/in/zengjinchen>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cchchoi1986/hikehk/pull/23/files#r24822969>.
>